src/lib.rs

```diff
@@ -25,6 +25,7 @@
 pub mod arithmetic;
 pub mod circuit;
 pub use pasta_curves as pasta;
+pub mod multicore;
```

Comment:
This doesn't need to be public:
```suggestion
mod multicore;
```